modules/ROOT/pages/secure-application-properties.adoc

```diff
@@ -16,13 +16,13 @@ CloudHub supports *safely hidden application properties*, where the name of a pr
 Internally, safely hidden application properties are encrypted and stored in a MuleSoft managed CloudHub properties database, which is encrypted per user organization.
 ====
 
-Safely hidden application properties are a feature that is currently only available for setting via the Runtime Manager UI for applications deployed to CloudHub workers. For applications that you deploy through any other xref:deployment-strategies.adoc[deployment strategy], you need to use secure property placeholder files bundled with your application zip file.
+For applications deployed to CloudHub workers, you must use the Runtime Manager UI to set hidden application. For applications that you deploy through any other xref:deployment-strategies.adoc[deployment strategy], you need to use secure property placeholder files bundled with your application ZIP file.
```

Comment:
"your application ZIP file" > "your application's `.zip` file" is better

Reply:
Salesforce style is ZIP.  
But I agree with adding the 's